manifests/init.pp

```diff
@@ -72,6 +72,14 @@
 #                                  (30 or 30s), minutes (30m), hours (6h), days (2d),
 #                                  or years (5y).
 #
+# $autosign::                      Eanble or disable autosign or change location
+#                                  of autosign.conf or autosign script.  If this
+#                                  is set to a script, make sure that script
+#                                  considers the content of autosign.conf, as it
+#                                  might break Foreman functionality if it
+#                                  doesn't.
+#                                  type:string or boolean
```

Comment:
I suppose this should be a new type (stroolean?) but IIRC we tried to avoid having parameters like this, my preference would be adding new parameter or in the worst case keeping it as string and treat 'true' and 'false' strings explicitely